compiler-core/src/erlang/tests/bit_arrays.rs

```diff
@@ -314,3 +314,19 @@ pub fn main() {
 "
     );
 }
+
+#[test]
+fn unicode_character_encoding_in_bit_array_pattern_segment() {
+    assert_erl!(
+        "
+pub fn main() -> Nil {
+  let foo = <<\"\\u{00A9}foo\":utf8>>
```

Comment:
It's a convention in the compiler to use `wibble`, `wobble`, `woo` instead of `foo`, `bar`, `baz`. Could you update the names here?

Reply:
Can do! Sorry, old habits haha